### PR TITLE
[formrecognizer] Regenerate with new swagger

### PR DESCRIPTION
**Pull request by catalinaperalta**
Regenerate with 2022-08-31 swagger. Remove kind property on DocumentPage. Update internal name references in the convenience layer. Update tests and recordings.

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- Removed `kind` property on `DocumentPage`.
```
Comment (catalinaperalta) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:8]: Should the changelog also mention that the only supported version going forward in 2022-08-31?

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2022_08_31/models/_form_recognizer_client_enums.py [hunk 2]
```diff
@@ -33,12 +33,6 @@ class ContentType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     APPLICATION_OCTET_STREAM = "application/octet-stream"
     #: Content Type 'application/pdf'.
     APPLICATION_PDF = "application/pdf"
-    #: Content Type 'application/vnd.openxmlformats-officedocument.presentationml.presentation'.
-    APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_PRESENTATIONML_PRESENTATION = "application/vnd.openxmlformats-officedocument.presentationml.presentation"
-    #: Content Type 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'.
-    APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_SPREADSHEETML_SHEET = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
-    #: Content Type 'application/vnd.openxmlformats-officedocument.wordprocessingml.document'.
-    APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_WORDPROCESSINGML_DOCUMENT = "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2022_08_31/models/_form_recognizer_client_enums.py:41]: Did we ever document support for these content-types that are being removed in this version?

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py [hunk 3]
```diff
@@ -78,7 +78,14 @@ async def test_business_card_multipage_pdf(self, client):
...
         assert len(business_card.fields.get("Addresses").value) == 1
-        assert business_card.fields.get("Addresses").value[0].value == "2 Kingdom Street\nPaddington, London, W2 6BD"
+        assert business_card.fields.get("Addresses").value[0].value.house_number == "2"
+        assert business_card.fields.get("Addresses").value[0].value.po_box == None
+        assert business_card.fields.get("Addresses").value[0].value.road == "Kingdom Street"
+        assert business_card.fields.get("Addresses").value[0].value.city == "London"
+        assert business_card.fields.get("Addresses").value[0].value.state == None
+        assert business_card.fields.get("Addresses").value[0].value.postal_code == "W2 6BD"
+        assert business_card.fields.get("Addresses").value[0].value.country_region == None
+        assert business_card.fields.get("Addresses").value[0].value.street_address == "2 Kingdom Street"
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py:88]: I'm confused how this test was passing before -- it looks like a breaking change? did they just start returning something new here in this api version?

File: sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ multiapi: true
 batch:
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md:28]: you can add `python3-only: true` under the basic config so autorest doesn't generate models for python 2.

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 1 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:8]: We should definitely mention that this version of the library defaults to the GA version

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2022_08_31/models/_form_recognizer_client_enums.py (hunk 2 above)
Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2022_08_31/models/_form_recognizer_client_enums.py:41]: No I already checked 👍 but I'll make sure to do a full docs pass again soon.

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py (hunk 3 above)
Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py:88]: They hadnt made this change for all prebuilts. Same as currency value.

Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py:88]: Got it, hopefully it's consistent now across prebuilts

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py (hunk 3 above)
Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py:88]: Added an issue to check this: https://github.com/Azure/azure-sdk-for-python/issues/25786

**Pull request merged**